MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,452 @@
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Date Picker
+</td>
+<td>
+{{'[DataFormDatePickerEditor]'| markdownify }}
+</td>
+<td>
+The DateTime, DateTimeOffset, DateOnly property and the property with [DataType(DataType.Date)] and [DataType(DataType.DateTime)] attributes.
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Time Picker
+</td>
+<td>
+{{'[DataFormTimePickerEditor]'| markdownify }}
+</td>
+<td>
+The TimeSpan, TimeOnly property and the property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+{{'[DataFormPickerEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+{{'[DataFormComboBoxEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+{{'[DataFormAutoCompleteEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutocomplete](https://help.syncfusion.com/xamarin/autocomplete)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+{{'[DataFormRadioGroupEditor]'| markdownify }}
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+</table>
+
+## Changing editor for type
+
+By default, the editors will be loaded based on the previous table. To change the editor for any type, use the [RegisterEditor] method and specify the type and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.RegisterEditor(typeof(string), DataFormEditorType.MultilineText);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `MultilineText` editor will be loaded for the string type instead of `Text` editor.
+
+## Changing editor for property
+
+To change the editor for any property, use the [RegisterEditor] method and specify the property name and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.RegisterEditor("On"), DataFormEditorType.Switch);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the Switch editor will be loaded for the `On` property (bool type) instead of `CheckBox` editor.
+
+## Creating new custom editor
```

Comment:
Remove **Creating new custom editor** topic from here, it will be included in editors topic